python/vrp.py

```diff
@@ -0,0 +1,94 @@
+from mage.geography import create_distance_matrix, LATITUDE, LONGITUDE
+from mage.constraint_programming import VRPConstraintProgrammingSolver
+from typing import Dict, List
+
+import mgp
+
+
+__distance_matrix = None
```

Comment:
It might be useful to stress test this. Keeping the distance matrix in memory for the whole time of execution might be pretty expensive.

Reply:
We can see with buda how to approach this. Or have a maximum matrix size after which it deletes itself so it does not stay in memory the whole time.